src/Resources/config/commands.yml

```diff
@@ -1,32 +1,26 @@
 services:
+    _defaults:
+        autoconfigure: true
```

Comment:
Can we add a `# Automatically tags commands as console.command` here and whenever we add something new that uses the `autoconfigure` extend this comment? Just for future reference and transparency :)